model.py

```diff
@@ -8069,6 +8069,10 @@ def add_entry(self, edition, annotation=None, first_appearance=None,
             entry.license_pool = edition.license_pool
         if featured is not None:
             entry.featured = featured
+
+        if was_new:
+            self.updated = datetime.datetime.utcnow()
+            _db.commit()
```

Comment:
I'm not convinced this commit() is necessary. If you want it for symmetry with the commit() in remove_entry(), it should always happen, not just when the entry is new.